config/gulp/build.js

```diff
@@ -4,18 +4,29 @@ var del = require('del');
 var pkg = require('../../package.json');
 var runSequence = require('run-sequence');
 var spawn = require('cross-spawn');
+var rename = require('gulp-rename');
 
 gulp.task('clean-all', function () {
   return del([
     './static/assets/**/*',
   ]);
 });
 
+gulp.task('move-state-files', function () {
```

Comment:
@theresaanna I think so, Maybe we can use Hugo to generate the JSON for you and use Bloodhound to cache it locally for newer browsers?

Essentially, [using something similar to this](http://gohugo.io/extras/datadrivencontent/) that would compile a `go template` into a `json` file.